app/models/conference.rb

```diff
@@ -12,14 +12,18 @@ class Conference < ActiveRecord::Base
   scope :ordered, ->(sort = {}) { order([sort[:order] || 'starts_on, name', sort[:direction] || 'asc'].join(' ')) }
   scope :in_current_season, -> { where(season: Season.current) }
 
+  def date_range
+    @date_range ||= DateRange.new(start_date: starts_on, end_date: ends_on).display
```

Comment:
If the `DateRange#display` is to be renamed to `#to_s` (see other comment), I would return the `DateRange` instance here, not the string. It's just a minor thing, but to me having the method match a (snake_cased) Ruby class name and also semantically _meaning_ the same thing but returning a string is a bit confusing.